packages/smooth_app/lib/pages/scan/scan_product_card_loader.dart

```diff
@@ -21,9 +20,6 @@ class ScanProductCardLoader extends StatelessWidget {
         final BuildContext context,
         final AsyncSnapshot<Product?> snapshot,
       ) {
-        if (snapshot.connectionState != ConnectionState.done) {
-          return SmoothProductCardLoading(barcode: barcode);
```

Comment:
Just to be sure @AshAman999, is the loading card still visible while the content is loading?

Reply:
It's still showing up, Is this what you meant to ask @g123k ??

https://user-images.githubusercontent.com/57723319/182381141-32ebf7b3-93eb-4630-b5c0-918b7dd9dff7.mp4